content/docs/searching.md

```diff
@@ -207,98 +207,94 @@ The **Search results** window by default appears docked at the bottom of the Not
 
-Here's a more detailed description of what happens for *RightClick* > **Copy Selected Line(s)**:
+More precisely:
 
-First, if the user makes a selection of text in the **Search results** window and copies it this way, only the lines of text touched (even partially) by the selection are part of the copy.  All other text with information about the search (pathname, line number, etc.) is *not* copied, even if part of the selection.  Secondly, if there is no active selection when the *RightClick* > **Copy Selected Line(s)** is invoked, results depend upon what exactly is under the mouse cursor during the *RightClick*:
-
-| *RightClick* item     | What gets copied when *RightClick* > **Copy Selected Line(s)** is run |
+| `RightClick` item     | What gets copied when **`RightClick` > Copy Selected Line(s)** is run |
 |-----------------------|-----------------------------------------------------------------------------------------------------|
-|a line with line # info|the entire line of the *RightClick* but without line number text                                     |
+|a line with line # info|the entire line of the `RightClick` but without line number text                                     |
 |a pathname header line |all the lines for that single file without pathname or line number text                              |
 |a "search" header line |all the lines for that search (1 or more files) without search header, pathname or line number text  |
 
-*Tip*:  It is possible to select and copy a rectangular selection of data from the **Search results** window.  This is done using the usual Shift+Alt+arrow keys or by holding Alt+LeftClick and dragging with the mouse.  This is really only practical when using the *Ctrl+c* method of copying; *RightClick* > **Copy Selected Line(s)** only copies entire lines, and this copy will only copy the single full line at the top/bottom of the column block.
+*Tip*:  It is possible to select and copy a rectangular selection of data from the **Search results** window.  This is done using the usual `Shift+Alt+arrow keys` or by holding `Alt+LeftClick` and dragging with the mouse.  This is really only practical when using the `Ctrl+C` method of copying; **`RightClick` > Copy Selected Line(s)** doesn't work this way.
 
-There is a capability to copy a list of the files that contained hits from past searches (v8.0.0 and later).  The **Copy Pathname(s)** context menu command will copy to the clipboard the full pathnames of all files appearing in **Search results**.
+There is a capability to copy the list of files containing hits from past searches (v8.0.0 and later).  The **Copy Pathname(s)** context menu command copies to the clipboard the full pathnames of all files appearing in **Search results**.
 
 ##### Other commands
 
 There are some commands that don't need a lot of explanation; these are:
 
-* **Collapse all**
-* **Uncollapse all**
+* **Fold all**
+* **Unfold all**
```

Comment:
Please rephrase
> * **Fold all** (previously called **Collapse all** before v8.4.6)
> * **Unfold all** (previously called **Collapse all** before v8.4.6)

When behavior has changed in "recent" versions, we need to describe both behaviors, because this manual is valid for all recent versions.  (Currently, "recent" means v8.0 or newer.)

Reply:
Noted.

Reply:
> Oh, also, how about we delete the sentence above that? Say these items don't need a lot of explanation and then proceed to explain them seems contradictory...

Good idea.